pom.xml

```diff
@@ -77,8 +77,8 @@
       <url>https://oss.sonatype.org/content/repositories/snapshots</url>
     </repository>
     <repository>
-      <id>continuuity-public</id>
-      <url>https://repository.continuuity.com/content/repositories/public</url>
+      <id>cask-public</id>
+      <url>https://repository.cask.co/content/repositories/public</url>
```

Comment:
Can you check with @dereklwood do we still have this? Also, I think we should remove this `repository` from the pom